keras_rcnn/layers/object_detection/proposal_target.py

```diff
@@ -0,0 +1,60 @@
+import keras.backend
+import keras.engine
+import tensorflow
+
+import keras_rcnn.backend
+
+
+class ProposalTarget(keras.engine.topology.Layer):
+    def __init__(self, **kwargs):
+        self.allowed_border = 0
+        self.clobber_positives = False
+        self.negative_overlap = 0.3
```

Comment:
Yeah that'd definitely be a step up. I figured lets leave it hardcoded for now until we have a place to configure everything nicely. Regardless, `__init__` arguments are better.